services/postgres/alembic/versions/2020-09-03T095523_add_task_run_name.py

```diff
@@ -0,0 +1,26 @@
+"""
+Add task run name
+
+Revision ID: e148cf9f1e5b
+Revises: c1f317aa658c
+Create Date: 2020-09-03 09:55:23.155317
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import JSONB, UUID
+
+
+# revision identifiers, used by Alembic.
+revision = "e148cf9f1e5b"
+down_revision = "70528cee0d2b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("task_run", sa.Column("name", sa.String))
```

Comment:
What's the need for a task_run name? IIRC this existed a while ago but was removed because it was an unnecessary piece of information

Reply:
We removed randomly-generated ones that we used to have, but there have been user requests for setting task run names, for example for dynamically generated mapped children.

Reply:
For the dynamically generated mapped children do you mean users are going to come in and name each of those task runs individually or adopt something like https://github.com/PrefectHQ/prefect/pull/2974 (which doesn't yet have a nice implementation)?

Reply:
The idea would be that if they provided a custom callable for producing a name, at the beginning of each run we could call this API route for them and update the name - so yea, the hope is that we can update that PR to simply call this route at the appropriate time